### ISSUE TITLE
Setting up new backup to folder with existing backups, removes all earlier backups

### ISSUE THREAD

**Issue by Foshitz** [JosefNemec/Playnite#3255]
### Bug Description

When reinstalling Playnite and setting up a new automatic backup to  folder with existing backup files from previous installation, the new installation removes all older backups and replaces it with the first and subsequent backups from the folder.

### Diagnostics ID

52fa0e36-ee37-4ebd-b188-235485cf445a

**Comment by JosefNemec** [comment]
This is based on a number of rotating backups you have set in settings -> backup -> rotating backups.

**Comment by Foshitz** [comment]
No. Rotating backups keeps the total number of available backups up to 11 files, with the oldest files getting deleted when a new backup is created.

During this reinstall, all my previous backups from the earlier installation got deleted and replaced by 1 backup from the new installation because I pointed the new Playnite-installation to store the backups in the same folder.
I hadn't activated rotating backups yet.

**Comment by JosefNemec** [comment]
Let me get this clear:
- you had a folder with rotating backups
- you installed Playnite with new default settings
- you pointed it to backup to a folder with existing rotating backup, but you didn't enable rotating backups
- existing rotating backup have been deleted

Is this correct? If so, it sounds like a correct behavior, because you told Playnite to backup into a specific folder but without keeping rotating backups, so Playnite made sure there's only one backup kept.

**Comment by Foshitz** [comment]
That is correct! I just assumed it would ignore the old files, since they were from a earlier installation. I wasn't prepared for them to be deleted without warning.
If this is the correct behavior, you might want to add some info about this in the Backup-settings. It should at least be warned about this before we apply the settings and confirm to restart Playnite.

**Comment by bonanza465** [comment]
I'll add to this issue something that I just observed:

1) I have an automatic backup rotation that has been running for a few months, I had total 4 prev backups in a folder due to automatic backup
2) I performed a manual backup from Playnite
3) All prev backups have been deleted from the folder

This doesn't involve multiple installations of Playnite.
